### PR TITLE
API cpu fields description

### PR DESCRIPTION
**Pull request by OhmSpectator**
Add a description for the CPU-related fields of the VM config

### CODE REVIEW

File: api/proto/config/vm.proto [hunk 1]
```diff
@@ -26,11 +26,21 @@ message VmConfig {
   string ramdisk = 2;
   uint32 memory = 3;
   uint32 maxmem = 4;
+  // The amount of the VCPUs in the VM.
   uint32 vcpus = 5;
+  // Currently is not handled by EVE.
```
Comment (giggsoff) [inline api/proto/config/vm.proto:31]: Seems, you are right, we do not propagate the field from config in parseAppInstanceConfig function. It is quite strange, should we modify it in https://github.com/lf-edge/eve/pull/2937 while it is open?

**Review by eriknordmark — APPROVED** [review]
LGTM but Petr's comment/question is something to consider.

File: api/proto/config/vm.proto (hunk 1 above)
Comment (OhmSpectator) [reply api/proto/config/vm.proto:31]: Ok, will do.

**Comment by OhmSpectator** [comment]
As we have already merged the previous PR, i added a commit with the maxcpus parameter parsing here.
Fix the comments in the code accordingly

**Review by eriknordmark — APPROVED** [review]
LGTM

**Comment by OhmSpectator** [comment]
The ZFS Eden tests failed. The ext4 ones are ok meanwhile. Hopefully, it's not related to the code changes introduced in the PR.

**Pull request merged**